server/routes/api.js

```diff
@@ -607,7 +607,7 @@ router.route('/messageSearch')
               qb2.select('id').from('capcodes').where('agency', agency).where('ignore', 0);
           })
           if (alias != '')
-            qb.where('alias.id',alias);
+            qb.where('alias_id',alias);
```

Comment:
should this be `messages.alias_id`? can't remember the data structure here so just going on context of the other `qb.where` things

Reply:
alias.id is not the valid column name 
`[pmon]  2021-01-23 20:27:17  error : select `messages`.*, `capcodes`.`alias`, `capcodes`.`agency`, `capcodes`.`icon`, `capcodes`.`color`, `capcodes`.`ignore` from `messages` left join `capcodes` on `capcodes`.`id` = `messages`.`alias_id` where `alias`.`id` = 'RFS HEART BEAT' order by `messages`.`timestamp` desc - SQLITE_ERROR: no such column: alias.id`

Reply:
oh sorry, I miss read
let me check

Reply:
Fixed in last commit